core/src/main/java/org/polypheny/db/algebra/enumerable/EnumUtils.java

```diff
@@ -239,21 +256,8 @@ static Type fromInternal( Type type ) {
     }
 
 
-    static Type toInternal( AlgDataType type ) {
-        switch ( type.getPolyType() ) {
-            /*case DATE:
-            case TIME:
-                return type.isNullable() ? Integer.class : int.class;
-            case TIMESTAMP:
-                return type.isNullable() ? Long.class : long.class;*/
-            default:
-                return null; // we don't care; use the default storage type
-        }
-    }
-
-
     static List<Type> internalTypes( List<? extends RexNode> operandList ) {
-        return Util.transform( operandList, node -> toInternal( node.getType() ) );
+        return Util.transform( operandList, node -> null );
```

Comment:
This looks wrong...

Reply:
In this case we previously called a different method, which set all values to null, which then uses the default storage type. I removed the method and annotated the callees.